desk/app/forums.hoon

```diff
@@ -113,7 +115,44 @@
     [~ %sss %behn @ @ @ %forums %updates @ ~]  [(behn:da-forums |3:wire) this]
   ==
 ::
-++  on-peek   _~
+++  on-peek
+  |=  =path
+  ^-  (unit (unit cage))
+  ?+    path  [~ ~]
+      [%x %meta ~]
+    ``[%noun !>(metadata:rock:(rear ~(val by read:du-forums)))]
+    ::
+      [%x %posts ~]
+    =+  rocks=~(val by read:du-forums)
+    :*  ~  ~  %noun  !>
+      ?~  rocks  ~
+      =+  rockish=(rear rocks)
+      =+  deebee=database.rock.rockish
+      ::  FIXME: Foul, but a simple way to get the mold from the `sur` file.
+      =+  post-mold=$:(post-id=@ parent-id=@ child-ids=[%s p=(set @)] votes=[%m p=(map @p term)] history=[%b p=((mop @da ,[who=@p content=@t]) gth)] ~)
+      %-  turn
+      :_  |=(row=row.nectar !<(post-mold [-:!>(*post-mold) row]))
+      =<  -
+      %+  ~(q db.nectar-lib deebee)  %forums
+      ^-  query.nectar
+      [%select %board-name-posts %n ~]
+    ==
+      [%x %threads ~]
+    =+  rocks=~(val by read:du-forums)
+    :*  ~  ~  %noun  !>
+      ?~  rocks  ~
+      =+  rockish=(rear rocks)
+      =+  deebee=database.rock.rockish
+      ::  FIXME: Foul, but a simple way to get the mold from the `sur` file.
+      =+  thread-mold=$:(post-id=@ reply-ids=[%s p=(set @)] title=@t tags=[%s p=(set term)] ~)
```

Comment:
can't we grab the post schema from `forum` like `thread:forum`? Same for the post mold above

Reply:
Yes, good call; this will be fixed in the next commit.